.github/workflows/pull_request.yml

```diff
@@ -68,9 +59,12 @@ jobs:
     name: Build
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         # With a new supported Kubernetes version we should adjust the version
-        kubever: [ "v1.17.0", "v1.18.0", "v1.19.0", "v1.20.0" ]
+        # See https://kubernetes.io/releases for the current releases
+        kubever: [ "v1.19.0", "v1.20.0", "v1.21.1" ]
+        fdbver: ["6.2.30", "6.3.15"]
```

Comment:
I think it makes sense to test at least these version (we could also test `6.1` but I think `6.2` and `6.3` is fine.